dev-docs/RFCs/v7.x/gpu-grid-layer.md

```diff
@@ -0,0 +1,40 @@
+# RFC: GPU Acceleration in GridLayer
+
+* **Authors**: Ravi Akkenapally
+* **Date**: May 2019
+* **Status**: **Draft**
+
+
+## Abstract
+
+GPU accelerated aggregation is currently available in `ScreenGridLayer` and `ContourLayer`. In 6.3 release, `GPUAggregator` is enhanced to support aggregation of multiple weights required for `GridLayer`. This RFC proposes adding GPU accelerated aggregation to `GridLayer`.
+
+
+## Fallback cases
+
+There are several features of `GridLayer` that can't be supported by GPU Aggregation.
+
+* `Lower and higher Percentile` : When these props are set to a non zero value, grid cells are sorted based on their aggregated values and some of the begining and ending cells are ignored (not rendered) based on specified percentile value. On GPU, sorting is not possible.
```

Comment:
Kept thinking about this. While filtering on the "aggregated" values may seem a bit more intuitive, this feature is typically used to help remove a few "outliers" in the input data that skew things too much.

So if we redefined these to filter the input data, it could still work, right? Maybe worth proposing, bringing up in audit discussions?